### PR TITLE
#837 responsive course overview

### LINKED ISSUE #837: Responsive Course Overview
# User Story

### As a: 
user of the geli plaform.

### I want:
to use the platform on mobile devices while still having a good design and readablitly.

## Acceptance criteria:
- [ ] The course overview is responsive

## Additional info:

image
image

## Estimate:
4h

### LINKED ISSUE #868: collapse of units in student view
# User Story

### As a: 
student

### I want:
to be able to collapse units of a lecture

### so that:
I can navigate/find specific topics easyer -- similar to the edit view of the teacher.

## Acceptance criteria:
- [ ] collapse all units of a lecture 
- [ ] expand a single unit of a lecture

## Estimate:
2h

### PR DESCRIPTION
**Pull request by lukas-schardt**
## Description:

Closes #837, closes #868 

## Improvements
- 

## Known Issues:
_NONE_

### CODE REVIEW

File: app/webFrontend/src/app/course/course-detail/download-course-dialog/download-course-dialog.component.scss [hunk 1]
```diff
@@ -49,3 +49,20 @@
 .radio-buttons .radio-button:nth-child(n+2){
   margin-left: 20px;
 }
+
+@media screen and (max-width: 475px) {
```
Comment (dboschm) [inline app/webFrontend/src/app/course/course-detail/download-course-dialog/download-course-dialog.component.scss]: I have a question: What does `@media screen` and/or `@media only screen` do?

File: app/webFrontend/src/app/course/course-container/course-container.component.scss [hunk 2]
```diff
@@ -44,3 +44,9 @@
   height: 36px;
   width: 36px;
 }
+
+h1.small {
+  @media only screen and (max-width: 475px) {
+    font-size: 20px;
```
Comment (dboschm) [inline app/webFrontend/src/app/course/course-container/course-container.component.scss]: Do we not have the convention of not using px, and use em instead?

**Comment by danielkesselberg** [comment]
@dboschm :ping_pong: 

image

File: CHANGELOG.md [hunk 3]
```diff
@@ -85,6 +85,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Sentry reporting for missing translations. [#858](https://github.com/geli-lms/geli/issues/858)
 - Migration for `visible` field. [#890](https://github.com/geli-lms/geli/pull/890)
 - Migration for `fileUnitType` field. [#907](https://github.com/geli-lms/geli/pull/907)
+- Fixed error on clicking on notification. [#916](https://github.com/geli-lms/geli/issues/916)
+- A collapse/expand button to units [#868](https://github.com/geli-lms/geli/issues/868)
```
Comment (danielkesselberg) [inline CHANGELOG.md]: Add a dot at the end of the sentence

File: CHANGELOG.md [hunk 4]
```diff
@@ -106,6 +109,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Update `README.md` with latest information. [#845](https://github.com/geli-lms/geli/pull/845)
 - Exit build when no change to `CHANGELOG.md`. [#880](https://github.com/geli-lms/geli/pull/880)
 - Use deploy token to push geli-docs. [#851](https://github.com/geli-lms/geli/issues/851), [#900](https://github.com/geli-lms/geli/pull/900), [#902](https://github.com/geli-lms/geli/pull/902)
+- Responsiveness of course overview [#837](https://github.com/geli-lms/geli/issues/837)
```
Comment (danielkesselberg) [inline CHANGELOG.md]: Add a dot at the end of the sentence

File: app/webFrontend/src/app/course/course-container/course-container.component.scss (hunk 2 above)
Comment (danielkesselberg) [reply app/webFrontend/src/app/course/course-container/course-container.component.scss]: https://engageinteractive.co.uk/blog/em-vs-rem-vs-px i couldn't care less about em, rem and px but we use em in most places so i would suggest to use em here as well :see_no_evil:

**Comment by danielkesselberg** [comment]
**Before**
image
image
image

**After**

localhost_4200_course_5bedba1ca5c1fe3bd6069cf1_overview
localhost_4200_course_5bedba1ca5c1fe3bd6069cf1_overview 1
localhost_4200_course_5bedba1ca5c1fe3bd6069cf1_overview 2

**Pull request merged**